sound/soc/intel/common/Makefile

```diff
@@ -18,5 +18,5 @@ snd-soc-acpi-intel-match-objs := soc-acpi-intel-byt-match.o soc-acpi-intel-cht-m
 obj-$(CONFIG_SND_SOC_INTEL_SST) += snd-soc-sst-dsp.o snd-soc-sst-ipc.o
 obj-$(CONFIG_SND_SOC_ACPI_INTEL_MATCH) += snd-soc-acpi-intel-match.o
 
-snd-soc-intel-sof-ssp-common-objs += soc-acpi-intel-ssp-common.o
-obj-$(CONFIG_SND_SOC_INTEL_SOF_SSP_COMMON) += snd-soc-intel-sof-ssp-common.o
+snd-soc-acpi-intel-ssp-common-objs += soc-acpi-intel-ssp-common.o
```

Comment:
renaming modules is a hassle, but in this case you're lucky that this is not used by sof-test, so there's no harm in renaming...